### PR TITLE
add openapi spec

### PR DESCRIPTION
**Pull request by beav**

### CODE REVIEW

File: drift/openapi/api.spec.yaml [hunk 1]
```diff
@@ -0,0 +1,36 @@
+---
+openapi: 3.0.1
+info:
+  version: "0.01"
```
Comment (ehelms) [inline drift/openapi/api.spec.yaml]: 0.01? Not seen that syntax, or is this a typo?

Comment (beav) [reply drift/openapi/api.spec.yaml]: nah, but I could put 0.1 in there or something else. Basically any number < 1 is ok:)

Comment (ehelms) [reply drift/openapi/api.spec.yaml]: But it's usually just an X.Y versioning scheme?

Comment (beav) [reply drift/openapi/api.spec.yaml]: ya you are right, I got the numbers swapped in my head https://semver.org/#how-should-i-deal-with-revisions-in-the-0yz-initial-development-phase

**Pull request merged**